Web/coolprop/wrappers/SMath/index.rst

```diff
@@ -38,13 +38,13 @@ Native Wrapper
     
   This will inject the version number in the appropriate format into a few template files using CMake
   
-3. Open the VS2010 solution file in ``wrappers/SMath``.  More recent versions of Visual Studio should be able to convert this project, including VS2013.
+3. Open the solution file in ``build`` directory.
 
-4. Make sure the mode is set to Release (not Debug).  Build the project, the generated and copied files will be in ``bin/Release``.
+4. Make sure the mode is set to Release (not Debug).  Build the project, the generated and copied files will be in ``build/wrappers/SMath/coolprop_wrapper/bin/Release``.
```

Comment:
It seems unlikely to me that the files would be in bin/Release relative to coolprop_wrapper, but you know your code better than I do...